container-chains/templates/simple/runtime/src/lib.rs

```diff
@@ -371,6 +375,7 @@ parameter_types! {
 impl pallet_sudo::Config for Runtime {
     type RuntimeCall = RuntimeCall;
     type RuntimeEvent = RuntimeEvent;
+    type WeightInfo = ();
```

Comment:
we should probably bring the weights calculated by substrate

Reply:
I thought that `()` is equivalent to the default weights, which are the ones calculated by substrate?

Reply:
I dont know for sure, I always thought it does not apply any weight.

Reply:
They are equivalent, but it says that `()` is for backwards compatibility and tests, so maybe we should change it in the future